src/mocks/constants.ts

```diff
@@ -186,3 +188,69 @@ export const MOCK_SUBFOLDER_NAME = "mock-subfolder"
 export const MOCK_RESOURCE_ROOM_NAME = "mock-resource-room"
 
 export const MOCK_RESOURCE_CATEGORY_NAME = "mock-resource-category"
+
+export const MOCK_ITEMS: EditedItemProps[] = [
+  {
+    type: ["page"],
+    name: "some file",
+    path: ["some", "thing"],
+    url: "www.google.com",
+    lastEditedBy: "asdf",
+    lastEditedTime: 129823104823094,
+  },
+  {
+    type: ["nav"],
+    name: "some file",
+    path: ["some", "thing"],
+    url: "www.google.com",
+    lastEditedBy: "asdf",
+    lastEditedTime: 129823104823094,
```

Comment:
Small nit: The timestamps should all be different so we can see if the sorting function works as intended.

Reply:
yep, this PR doesn't include sorting - the follow up PR does so this isn't an issue